lib/utils.js

```diff
@@ -505,6 +505,26 @@ function isPromiseLike(maybePromise) {
   return maybePromise && typeof maybePromise.then === 'function';
 }
 
+/**
+ * Applies collation to a given command.
+ *
+ * @param {object} [command] the command on which to apply collation
+ * @param {(Cursor|Collection)} [self]
```

Comment:
can we rename this something slightly more descriptive? even `target` would be preferable to `self` imho 😄